package-lock.json

```diff
@@ -7779,8 +8155,7 @@
                         },
                         "ansi-regex": {
                             "version": "2.1.1",
-                            "bundled": true,
-                            "optional": true
+                            "bundled": true
```

Comment:
I guess removing `"optional": true` is an accidental change.
Please make sure you're using fresh npm and node versions (mine are node v12.16.1, npm 6.13.4), clean node_modules and re-run npm install. No existing dependencies modification are expected in this file, only new lines added.

Commit which contains these changes is "Added worker copy depending on the environment variable value", so you can do interactive rebase and Edit it so that the fix was immediately applied to that commit.

Reply:
BTW, you can also use Edit during interactive rebase to apply small fixes and get rid of corresponding commit.